config/resources/besluit-domain.lisp

```diff
@@ -200,6 +200,8 @@
                                         :as "kind")
              (meeting                   :via      ,(s-prefix "dct:isPartOf")
                                         :as "main-meeting")
+             (piece                     :via      ,(s-prefix "besluitvorming:heeftNotulen")
```

Comment:
With the change to a subtype, shouldn't this be `minutes`?

Reply:
done in [d763353](https://github.com/kanselarij-vlaanderen/app-kaleidos/pull/392/commits/d76335356cb6c9e75ed4a0993e9a53dd7cb7c772)